generator/routing_index_generator.cpp

```diff
@@ -263,8 +267,58 @@ void CalcCrossMwmTransitions(string const & mwmFile, string const & mappingFile,
                              vector<CrossMwmConnectorSerializer::Transition<connector::TransitId>> & transitions)
 {
   CHECK(mappingFile.empty(), ());
-  NOTIMPLEMENTED();
-  // @todo(bykoianko) Filling |transitions| based on transit section should be implemented.
+  try
+  {
+    FilesContainerR cont(mwmFile);
+    CHECK(cont.IsExist(TRANSIT_FILE_TAG),
+          (TRANSIT_FILE_TAG, "should be generated before", TRANSIT_CROSS_MWM_FILE_TAG));
+    auto reader = cont.GetReader(TRANSIT_FILE_TAG);
+    ReaderSource<FilesContainerR::TReader> src(reader);
+
+    transit::FixedSizeDeserializer<ReaderSource<FilesContainerR::TReader>> numberDeserializer(src);
+    transit::TransitHeader header;
+    numberDeserializer(header);
+    CHECK(header.IsValid(), ("TransitHeader is not valid.", header));
+
+    transit::Deserializer<ReaderSource<FilesContainerR::TReader>> deserializer(src);
+    vector<transit::Stop> stops;
+    deserializer(stops);
+    CHECK(IsValidSortedUnique(stops), ("Transit stops are not valid. Mwm:", mwmFile));
+    src.Skip(header.m_edgesOffset - header.m_gatesOffset); // Skipping gates.
+    vector<transit::Edge> edges;
+    deserializer(edges);
+    CHECK(IsValidSortedUnique(edges), ("Transit edges are not valid. Mwm:", mwmFile));
+
+    auto const getStopIdPoint = [&stops] (transit::StopId stopId) -> m2::PointD const & {
```

Comment:
Wrong formatting - no need in space between capture list and parameter list.

Reply:
Fixed with clang format.